app/views/positions/index.html.haml

```diff
@@ -0,0 +1,24 @@
+%h1 Positions
+
+%h2
+  = link_to 'New Position', new_position_path, class: 'btn btn-primary'
+
+%table.table.table-striped.table-bordered.table-hover
+  %tr
+    %th Name
+    %th Union
+    %th.min-width Actions
+  - @positions.each do |position|
+    %tr
+      %td
+        = position.name
+      %td
+        = position.union.name
+      %td{class: 'min-width', align: 'center'}
+        = link_to (content_tag(:span, nil, class: 'glyphicon glyphicon-trash')+' Delete'),
```

Comment:
same here, use `button_to`

Reply:
I based my code off of umaps, the reason they were using a link_to is because button_to uses a <input> tag which doesn't render html within it, so it wasn't able to have the glyphicon.

Here's the best solution I could come up with:
``` ruby
= form_tag division, method: :delete, data: { confirm: 'Are you sure?' } do
  = button_tag(class: 'btn btn-xs btn-danger') do
    = content_tag(:span, nil, class: 'glyphicon glyphicon-trash')
    Delete
```
It looks identical to the original code using a link_to while rendering as a button in without css. However, it seems a little excessive. There might be a simpler way to do it that I'm not seeing.